src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices.Tests/Implementation/InstanceAppSITests.cs

```diff
@@ -111,6 +174,13 @@ private void InitializeMocks(HttpResponseMessage httpResponseMessage)
                     ItExpr.IsAny<CancellationToken>())
                 .ReturnsAsync(httpResponseMessage)
                 .Verifiable();
+
+
+            readStatushandlerMock.Protected()
+                .Setup<Task<HttpResponseMessage>>("SendAsync", ItExpr.Is<HttpRequestMessage>(p => p.RequestUri.ToString().Contains("readstatus?status=")),
```

Comment:
Instead of creating a new mock for every endpoint we want to verify. We could add a parameter to the InitializeMocks method.
```suggestion
                .Setup<Task<HttpResponseMessage>>("SendAsync", ItExpr.Is<HttpRequestMessage>(p => p.RequestUri.ToString().Contains(urlPath)),
```

Reply:
Lurt.. Done